config/production.js

```diff
@@ -10,4 +10,4 @@ module.exports = {
       }
     }
   }
-};
\ No newline at end of file
+};
```

Comment:
This still doesn't change it back to postgres..!

Reply:
👍